shipgen/EvtCalcGenerator.cxx

```diff
@@ -17,10 +17,10 @@ Bool_t EvtCalcGenerator::Init(const char* fileName) {
 // -----   Default constructor   -------------------------------------------
 Bool_t EvtCalcGenerator::Init(const char* fileName, const int firstEvent) {
   
-  fInputFile = TFile::Open(fileName, "read");
+  fInputFile = std::make_unique<TFile>(TFile::Open(fileName, "read"));
```

Comment:
I didn't see this earlier, but this constructs a copy of the file, instead of adopting the pointer. Not a big issue, but I think just creating a unique_ptr would be better.

Reply:
Implemented.